Dockerfile

```
@@ -2,7 +2,7 @@ ARG BASE_IMAGE_FULL
 ARG BASE_IMAGE_MINIMAL
 
 # Build node feature discovery
-FROM golang:1.17.2-buster as builder
+FROM golang:1.18-buster as builder
```

nit: let's continue to use a pinpointed version for reproducible builds

```suggestion
FROM golang:1.18.0-buster as builder
```